frontend/src/components/sdcCollection/SDCCollectionSummary.vue

```diff
@@ -160,36 +160,36 @@ export default {
   }
 };
 </script>
-    
-    <style scoped>
+
+<style scoped>
+.containerSetup{
+  padding-right: 10em !important;
+  padding-left: 10em !important;
+}
+
+@media screen and (max-width: 1200px) {
   .containerSetup{
-    padding-right: 10em !important;
-    padding-left: 10em !important;
-  }
-  
-  @media screen and (max-width: 1200px) {
-    .containerSetup{
-      padding-right: 3em !important;
-      padding-left: 3em !important;
-    }
-  } 
-  .border {
-    border: 2px solid grey;
-    border-radius: 5px;
-    padding: 35px;
-    margin-bottom: 2em;
-    margin-top: 2em;
-    width: 70%;
+    padding-right: 3em !important;
+    padding-left: 3em !important;
   }
```

Comment:
It's hard to tell what the changes were on this file. Is it just some whitespace that was changed?

Reply:
Good point John, I do realize it's hard to tell. I made some indents changes to the file according to linters. No other changes have been made otherwise

Reply:
The use of the `!important;` flag is pretty aggressive as well.  It means, "It must be this way, no exceptions!" and effectively breaks the "cascading" aspect of CSS.  If I may, I would encourage you to find another way to apply your changes.